web/views.py

```
@@ -2698,8 +2718,115 @@ def challenge_submit(request, week_number):
+    quiz = Quiz.objects.filter(
+        id = quiz_id
+    ).first()
+    
+    if not quiz:
+        print("No active quiz found.")
+        return render(request, "web/current_live_challenge.html", {"quiz": None})
+
+    user_submission = None
+    has_submitted = False
+    questions = quiz.questions.prefetch_related("options").all()
+    print("questions are:",questions);
```

Ruff (0.8.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)